stdlib/option.ml

```diff
@@ -20,7 +20,7 @@ let get = function Some v -> v | None -> invalid_arg "option is None"
 let map f o = match o with None -> None | Some v -> Some (f v)
 let flatten = function Some (Some _ as o) -> o | _ -> None
 let iter f = function Some v -> f v | None -> ()
-let fold f o acc = match o with Some v -> f v acc | None -> acc
+let fold none some = function Some v -> some v | None -> none
```

Comment:
Why the change to remove the accumulator?

Reply:
You can easily recover that, see discussion above.